lisa/sut_orchestrator/azure/arm_template.json

```diff
@@ -2,12 +2,6 @@
     "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
     "contentVersion": "1.0.0.0",
     "parameters": {
-        "storage_name": {
```

Comment:
What's useful of this file now? Should we start to delete it?

Reply:
I think so, let me raise another PR to delete it.